integration-tests/environments/react-native-test-app/harness/runner.js

```diff
@@ -139,7 +139,7 @@ async function run(spawnLogcat) {
     if (PLATFORM === "android") {
       // Start the log cat (skipping any initial pid from an old run)
       if (spawnLogcat) {
-        logcat.start("com.realmreactnativetests", true).catch(console.error);
+        logcat.start("com.microsoft.reacttestapp", true).catch(console.error);
```

Comment:
Technically no longer needed, but it does fix an issue of spawning logcat when running locally.